make/Makefile.dev

```diff
@@ -0,0 +1,60 @@
+DOCKER_REPO?=quay.io/openshiftio
+IMAGE_NAME?=git-service
+SHORT_COMMIT=$(shell git rev-parse --short HEAD)
+ifneq ($(GITUNTRACKEDCHANGES),)
+SHORT_COMMIT := $(SHORT_COMMIT)-dirty
+endif
+
+TIMESTAMP:=$(shell date +%s)
+TAG?=$(SHORT_COMMIT)-$(TIMESTAMP)
+
+DEPLOY_DIR:=deploy
+
+.PHONY: create-resources
+create-resources:
+	@echo "Logging using system:admin..."
+	@oc login -u system:admin
+	@echo "Creating sub resources..."
+	@echo "Creating CRDs..."
+	@oc create -f https://raw.githubusercontent.com/redhat-developer/devopsconsole-operator/master/deploy/crds/devopsconsole_v1alpha1_gitsource_crd.yaml
+	@echo "Creating Namespace"
+	@oc create -f $(DEPLOY_DIR)/namespace.yaml
+
+.PHONY: create-cr
+create-cr:
+	@echo "Creating Custom Resource..."
+
+.PHONY: build-image
+build-image:
+	docker build -t $(DOCKER_REPO)/$(IMAGE_NAME):$(TAG) -f Dockerfile.dev .
+	docker tag $(DOCKER_REPO)/$(IMAGE_NAME):$(TAG) $(DOCKER_REPO)/$(IMAGE_NAME):test
+
+.PHONY: deploy-apiserver-only
+deploy-apiserver-only:
+	@echo "Creating API Server..."
+
+.PHONY: clean-all
+clean-all:  clean-apiserver clean-resources
+
+.PHONY: clean-apiserver
+clean-apiserver:
+	@echo "Deleting API Server..."
+
+.PHONY: clean-resources
+clean-resources:
+	@echo "Deleting sub resources..."
+	@echo "Deleting Namespace"
+	@oc delete -f $(DEPLOY_DIR)/namespace.yaml || true
+	@echo "Deleting CRDs..."
+	@oc delete -f https://raw.githubusercontent.com/redhat-developer/devopsconsole-operator/master/deploy/crds/devopsconsole_v1alpha1_gitsource_crd.yaml || true
+
+.PHONY: deploy-apiserver
+deploy-apiserver: build build-image deploy-apiserver-only
+
+.PHONY: minishift-start
+minishift-start:
+	minishift start --cpus 4 --memory 8GB
+	-eval `minishift docker-env` && oc login -u system:admin
```

Comment:
you could also add `eval minishift oc-env` to install the `oc` command in the `$PATH`. Also, the syntax may be slightly different on macOS: `eval $(minishift oc-env) && eval $(minishift docker-env)`